Dynamic Data API guides (#2104)
spryker/spryker-docs: docs/scos/dev/glue-api-guides/202307.0/dynamic-data-api/how-to-guides/how-to-configure-dynamic-data-api.md

---
title: How to configure Dynamic Data API endpoints.
description: This guide shows how to configure the Dynamic Data API endpoints.

> last_updated: August 21, 2023

Reviewer: ```suggestion
last_updated: Aug 21, 2023
```

template: howto-guide-template
redirect_from: 
  - /docs/scos/dev/glue-api-guides/202304.0/dynamic-data-api/how-to-guides/how-to-configure-dynamic-data-api.html
---

This guide shows how to configure the Dynamic Data API endpoints.

To incorporate a new endpoint for interacting with entities in the database,

> add a corresponding row to the `spy_dynamic_entity_configuration` table or create a configuration file and install configuration data. For details, see [Dynamic Data API integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/glue-api/dynamic-data-api-integration.html)

Reviewer: ```suggestion
add a corresponding row to the `spy_dynamic_entity_configuration` table or create a configuration file and install configuration data. For details, see the [Configure Dynamic Data installation](#dynamic-data-install-configuration) section
```

This guide shows how to configure the Dynamic Data API endpoints.

In order to incorporate a new endpoint for interacting with entities in the database, 

> it is necessary to add a corresponding row to the `spy_dynamic_entity_configuration` table or needs to create a configuration file and import configuration data, see ([Dynamic Data API integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/glue-api/dynamic-data-api-integration.html#dynamic-data-import-configuration))

Reviewer 2: import -> install :)

> it is necessary to add a corresponding row to the `spy_dynamic_entity_configuration` table or needs to create a configuration file and install configuration data, see ([Dynamic Data Install Configuration](/docs/scos/dev/feature-integration-guides/{{page.version}}/glue-api/dynamic-data-api-integration.html#dynamic-data-install-configuration))

Reviewer: The section "dynamic-data-install-configuration" does not exist, so the link to the section does not work; it simply opens the Dynamic Data API integration guide.
Therefore, please, adjust the link and it's name accordingly.
```suggestion
add a corresponding row to the `spy_dynamic_entity_configuration` table or create a configuration file and install configuration data. For details, see [Dynamic Data Install Configuration](/docs/scos/dev/feature-integration-guides/{{page.version}}/glue-api/dynamic-data-api-integration.html#dynamic-data-install-configuration)
```

{% info_block infoBox %}

> Currently, the process entails manually executing SQL queries or importing via config file as there is no existing user interface (UI) feature in Spryker for it. 

Reviewer: don't use "via"
Use "because" instead of "as"
```suggestion
The process entails manually executing SQL queries or importing using a config file because there is no existing user interface (UI) feature in Spryker for it. 
```
